tests/main/security-dev-input-event-denied/task.yaml

```diff
@@ -93,16 +90,15 @@ execute: |
 
     echo "When the device-buttons plug is connected"
     snap connect test-snapd-event:device-buttons
+    udevadm trigger
```

Comment:
Do we need this? We do trigger the relevant subsystem on connect from snapd.

Reply:
(same wondering about settle in this test, seems we do this in the reloadRules of udev backend.

Reply:
We dont need that if we have the retry, I left that to make sure the events are processed. I'll make a try incrementing the retry time and removing this.
Thanks for the review

Reply:
I think it makes sense to remove settle/trigger then because they are normally not needed for the functionality and having them may hide issues.